src/systems/sensors/Sensors.cc

```diff
@@ -38,10 +38,12 @@
 
 #include "ignition/gazebo/components/Atmosphere.hh"
 #include "ignition/gazebo/components/Camera.hh"
+#include "ignition/gazebo/components/BoundingBoxCamera.hh"
```

Comment:
```suggestion
#include "ignition/gazebo/components/Atmosphere.hh"
#include "ignition/gazebo/components/BoundingBoxCamera.hh"
#include "ignition/gazebo/components/Camera.hh"
```
nit: alphabetical order